client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTableFooter/GroupedInvestigations/GroupedInvestigations.tsx

```diff
@@ -11,30 +11,46 @@ import GroupedInvestigationsForm from './GroupedInvestigationsForm/GroupedInvest
 import GroupedInvestigationsTable from './GroupedInvestigationsTable/GroupedInvestigationsTable';
 import GroupedInvestigationsFields from './GroupedInvestigationsForm/GroupedInvestigationsFields';
 
-const GroupedInvestigations: React.FC<Props> = ({ invetigationsToGroup, open, onClose }: Props) => {
+const GroupedInvestigations: React.FC<Props> = (props: Props) => {
 
-    const groupedInvestigation = invetigationsToGroup.find((investigation: InvestigationTableRow) => investigation.groupId !== null);
-    const title = groupedInvestigation ? 'הוספת חקירות לקיבוץ' : 'קיבוץ חקירות'
+    const { open, invetigationsToGroup, allGroupedInvestigations, onClose, fetchTableData, fetchInvestigationsByGroupId } = props
 
     const methods = useForm<GroupForm>({
         mode: 'all',
         resolver: yupResolver(validationSchema),
         defaultValues: {
-            [GroupedInvestigationsFields.REASON]: groupedInvestigation ?
-                { displayName: groupedInvestigation.groupReason, id: groupedInvestigation.reasonId } : null,
-            [GroupedInvestigationsFields.OTHER_REASON]: groupedInvestigation ? groupedInvestigation.otherReason : ''
+            [GroupedInvestigationsFields.REASON]: null,
+            [GroupedInvestigationsFields.OTHER_REASON]: ''
         }
     });
+    
+    const groupedInvestigation = useMemo(() => {
```

Comment:
Add Boolean type

Reply:
why boolean?

Reply:
Oops didn’t see that it is being used as an object 🙊